### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by thomas2001u**
_This is my second pull request, in my first request I accidentally did a request from the incorrect branch._
_The initial request can be found here: https://github.com/shakes76/PatternFlow/pull/465_
I didn't realise I made this mistake until today, I hope that's forgivable, thanks. Commits are dated before the due date.

This is the PatternFlow project submission for COMP3710 by Thomas Barton (s4641500). This includes a VQVAE with Pixel CNN trained and tested on the OASIS dataset. The algorithm trains using png slices of OASIS brains to create a seed for outputting slices based on an input gathered from encoding brains. Testing on this input can show a brain that looks either more healthy or unhealthy, based on the training dataset used. It can be used to gauge the health of a brain by comparing outputs with the VQVAE.

EDIT: Please also note that two README files have been committed. The one under XUE4645768 was automatically added for some reason. It is not mine so please disregard it.

### CODE REVIEW

**Comment by shakes76** [comment]
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate use and implementation (code seems to be there, but can't verify if working, no outputs or training loss) -5
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem (no results provided, no convergence plotted, no generations) -3
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage (no plots or outputs) -2
Module present
Commenting, some missing (see train.py for example) -1
No Data leakage (can't verify generations or data sets used) -2
Difficulty: Hard
## Commit Log
Meaningful commit messages, could be more descriptive -1
Progressive commits used, could not determine any commits before 21 Oct because of incorrect PRs -1
## Documentation
ReadMe minimal (no architecture, background etc.) -2
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
Feedback required, remove changes to other student files, resolve conflicts -2
Request Description good
LATE, but reason give is acceptable because original PR and commits were made in time

**Comment by thomas2001u** [comment]
Tried reverting changes to XUE4645768 folder by reverting to previous commit then force pushing that to PR. It's appearing to delete other students' files though? This sort of thing wasn't covered in the required git courses so I'm having trouble considering I only had half the day to figure this out.
